andela/the_spinners-frontend
# #171250893 User can logout

#### What does this PR do?
```
Add user logout
Correct bug in reset password reload
```
#### Description of Task to be completed?
```
As a logged in user, I should be able to logout
```
#### How should this be manually tested?
Switch to this branch 
run `npm run start-dev`
visit /logout
then you should be logged out.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[171250893](https://www.pivotaltracker.com/story/show/171250893)
#### Screenshots (if appropriate)
#### Questions:
